src/populateExchange.ts

```diff
@@ -167,7 +167,7 @@ export const extractSelectionsFromQuery = (
 export const addFragmentsToQuery = (
   schema: GraphQLSchema,
   query: DocumentNode,
-  typeFragments: TypeFragmentMap,
+  activeTypeFragments: TypeFragmentMap,
```

Comment:
:heavy_plus_sign: for this - We should probably do this for [here also](https://github.com/FormidableLabs/urql-exchange-graphcache/blob/8f0cbc326964ce0c090c4ba6b965285841834ad3/src/populateExchange.ts#L41)